lib/Models/ViewerMode.ts

```diff
@@ -24,6 +25,12 @@ export const MapViewers = Object.seal({
     terrain: false,
     label: "settingPanel.viewerModeLabels.Leaflet",
     available: true
+  },
+  cesium2d: {
+    viewerMode: ViewerMode.Cesium2D,
+    terrain: false,
+    label: "settingPanel.viewerModeLabels.Leaflet",
```

Comment:
The other entries have unique labels, so I'm guessing this entry shouldn't use the already used Leaflet-label.